### PR TITLE
Feature/accessibility

### PR DESCRIPTION
**Pull request by innaLebedyk**
Add special attributes to the html-components

### CODE REVIEW

File: src/components/sections/SectionsList/SectionsList.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ export const SectionsListComponent = ({ sections, navigateToQuestion }) => {
           <SectionsListItem
             key={index}
             isDummy={false}
+            arialLabel={section.title}
```
Comment (YuliaSavchuk) [inline src/components/sections/SectionsList/SectionsList.js]: arialLabel -> ariaLabel

Comment (innaLebedyk) [reply src/components/sections/SectionsList/SectionsList.js]: fixed it

File: public/lang/en.json [hunk 2]
```diff
@@ -24,5 +24,18 @@
     "continueCourse": "Continue course",
     "startSection": "Start section",
     "statementQuestionTrueText": "True",
-    "statementQuestionFalseText": "False"
+    "statementQuestionFalseText": "False",
```
Comment (YuliaSavchuk) [inline public/lang/en.json]: Please, check these texts with PO

File: src/components/treeOfContent/Header/Header.js [hunk 3]
```diff
@@ -26,7 +27,8 @@ export class Header extends Component {
 Header.propTypes = {
   courseTitle: PropTypes.string,
   collapsed: PropTypes.bool,
-  navigateToIndex: PropTypes.func.isRequired
+  navigateToIndex: PropTypes.func.isRequired,
+  ariaLabel: PropTypes.string
```
Comment (innaLebedyk) [inline src/components/treeOfContent/Header/Header.js]: removed prop - ariaLabel

File: src/components/treeOfContent/SectionTreeNode/SectionTreeNode.js [hunk 4]
```diff
@@ -37,8 +37,18 @@ export default class SectionTreeNode extends Component {
     const { title, isPassed, affectsProgress, children } = this.props;
     const { isExpanded } = this.state;
     return (
-      <SectionItem>
-        <Header onClick={this.toggleIsExpanded}>
+      <SectionItem ariaLabel={`Section ${title}`}>
+        <Header
+          ariaLabel={
+            isExpanded ? `${localize('closeSectionItems')}` : `${localize('openSectionItems')}`
```
Comment (YuliaSavchuk) [inline src/components/treeOfContent/SectionTreeNode/SectionTreeNode.js]: localize(isExpanded ? 'closeSectionItems' : 'openSectionItems')

**Pull request merged**